tests/functional/adapter/python_model_tests/test_logging.py

```diff
@@ -0,0 +1,43 @@
+from dbt.tests.util import run_dbt
+import pytest
+
+
+MODEL__LOGGING = """
+import logging
+
+import snowflake.snowpark as snowpark
+import snowflake.snowpark.functions as f
+from snowflake.snowpark.functions import *
+
+
+logger = logging.getLogger("dbt_logger")
+logger.setLevel(logging.INFO)
```

Comment:
It looks like if we move the logging to use the Python `logging` module, then it works inside of the model, leaving only the `TRACE_LEVEL` setting as a sticking point.

Reply:
It looks like this might have been a timing issue. Apparently it takes over a minute for the events to show up in the event table, so this might have been associated with a previous scenario. It appears that setting the logging from within the python model does not work, whether running via SQL, `.setLevel()`, or setting `.level`.